dataverse-webapp/src/main/webapp/metadataTab.xhtml

```diff
@@ -0,0 +1,42 @@
+<ui:composition xmlns="http://www.w3.org/1999/xhtml"
+                xmlns:h="http://java.sun.com/jsf/html"
+                xmlns:f="http://java.sun.com/jsf/core"
+                xmlns:ui="http://java.sun.com/jsf/facelets"
+                xmlns:p="http://primefaces.org/ui"
+                xmlns:jsf="http://xmlns.jcp.org/jsf"
+>
+
+    <div class="button-block tab-header margin-bottom text-right">
+        <p:link styleClass="btn btn-default btn-access" disabled="#{DatasetMetadataTab.datasetLocked}"
```

Comment:
I think that lock state can be changed during the lifetime of the view. On dataset page we have some refreshing code. When initially dataset is locked then in 1.5s intervals there is a check if lock is still present. However in this tab there always will be initial lock state.